src/Agent.Worker/TestResults/TestResultsServer.cs

```diff
@@ -46,7 +46,7 @@ public void InitializeServer(VssConnection connection)
            RunCreateModel testRunData,
            CancellationToken cancellationToken = default(CancellationToken))
         {
-            return await TestHttpClient.CreateTestRunAsync(projectName, testRunData, cancellationToken);
+            return await TestHttpClient.CreateTestRunAsync(testRunData, projectName, cancellationToken);
```

Comment:
This is precisely why I like using named parameters, it provides compile time checking for bugs like this :)